UnitTests/expected/Storyboards-AdditionalImport-Swift3.swift.out

```diff
@@ -0,0 +1,66 @@
+// Generated using SwiftGen, by O.Halligon — https://github.com/AliSoftware/SwiftGen
+
+import Foundation
+import UIKit
+import LocationPicker
+import SlackTextViewController
+
+// swiftlint:disable file_length
+// swiftlint:disable line_length
+// swiftlint:disable type_body_length
+
+protocol StoryboardSceneType {
+  static var storyboardName: String { get }
+}
+
+extension StoryboardSceneType {
+  static func storyboard() -> UIStoryboard {
+    return UIStoryboard(name: self.storyboardName, bundle: nil)
+  }
+
+  static func initialViewController() -> UIViewController {
+    guard let vc = storyboard().instantiateInitialViewController() else {
+      fatalError("Failed to instantiate initialViewController for \(self.storyboardName)")
+    }
+    return vc
+  }
+}
+
+extension StoryboardSceneType where Self: RawRepresentable, Self.RawValue == String {
+  func viewController() -> UIViewController {
+    return Self.storyboard().instantiateViewController(withIdentifier: self.rawValue)
+  }
+  static func viewController(identifier: Self) -> UIViewController {
+    return identifier.viewController()
+  }
+}
+
+protocol StoryboardSegueType: RawRepresentable { }
+
+extension UIViewController {
+  func performSegue<S: StoryboardSegueType>(segue: S, sender: AnyObject? = nil) where S.RawValue == String {
+    performSegue(withIdentifier: segue.rawValue, sender: sender)
+  }
```

Comment:
#194 is probably going to land soon. You may want to reflect what is in there as well and merge as it comes to the `performSegue:`

https://github.com/AliSoftware/SwiftGen/pull/194/files#diff-5eb3bf790669fb7ada0dfec0ac55ca5cR39